pkgs/development/tools/just/default.nix

```
@@ -20,22 +20,50 @@ rustPlatform.buildRustPackage rec {
     export USER=just-user
     export USERNAME=just-user
 
-    sed -i tests/integration.rs \
+    sed -i tests/*.rs \
```

```suggestion
    sed -i src/justfile.rs tests/*.rs \
```

Then the `sed -i src/justfile.rs` block can be removed.